### DISCUSSION TITLE
How to report the projects who spread toxic Russian propaganda?

### DISCUSSION THREAD

**Discussion by rshopa** [community/community#56299] [Other Feature Feedback, Questions, & Ideas]
I failed to find such a topic using the search, so, apologies for possible duplications.

From my experience, it has little effect even to report a user who posted an abusive comment, least say major projects, when it comes to the question of Russian claims over Ukrainian territory and the supporting far-right propaganda. Due to the GitHub nature, such things are often subtle and indirect, not easily spotted at the first look, while political rants are not encouraged.

Besides the repeated "report abuse" form, a public discussion on this forum or going viral in social media, _are there additional options to file a complaint about the aforementioned content?_ 

One example: I reported earlier the author of the popular [Natural Earth project](https://github.com/nvkelso/natural-earth-vector) that provides world map tools, for the systematical refusal to depict [Crimea as Ukrainian territory](https://en.wikisource.org/wiki/United_Nations_General_Assembly_Resolution_68/262) as a default. [Numerous issues regularly posted](https://github.com/nvkelso/natural-earth-vector/issues?q=is%3Aissue+crimea) by other users are [censored, closed without explanations with treats of blocking](https://github.com/nvkelso/natural-earth-vector/issues/791#issuecomment-1559664400). I consider such behaviour is disrespectful and deeply insulting towards Ukrainian GitHub community, but I doubt another report would not end up in a traditional formal refusal with references to "freedom of expessions" etc.

Thnk You for the eventual advices.

**Comment by ogoun** [comment]
You should study history. Crimea was both Russian and Ukrainian territory. At present it belongs to Russia, and on the maps it should belong to Russia. If in the course of any negotiations or military action it is ceded to Ukraine, then it will be displayed as Ukrainian territory. There is no need to mislead people and start political discussions. It is necessary to follow the facts. (if anything, I was born in Ukraine)

**Reply by doskiukas-max** [reply]
Fking russian liar
